includes/class-wc-stripe-webhook-handler.php

```diff
@@ -954,14 +952,11 @@ public function process_payment_intent_success( $notification ) {
 						]
 					);
 
-					do_action( 'wc_gateway_stripe_process_payment_intent_incomplete', $order );
-					return;
```

Comment:
This return caused the unlock payment to never ran. I removed it now it's not necessary given the switch logic has been refactored